README.md

```diff
@@ -48,6 +48,8 @@ While designing distributed systems are hard enough, testing them is even harder
 - [Dapper](http://static.googleusercontent.com/media/research.google.com/en//pubs/archive/36356.pdf), Google's large scale distributed-systems tracing infrastructure, this was also the basis for the design of open source projects such as [Zipkin](http://zipkin.io/), [Pinpoint](https://github.com/naver/pinpoint) and [HTrace](http://htrace.incubator.apache.org/).
 
 ## Courses
+- [Reliable Distributed Algorithms, Part 1] (https://www.edx.org/course/reliable-distributed-algorithms-part-1-kthx-id2203-1x-0), KTH Sweden
```

Comment:
I think there is a space between the braces causing markdown to improperly render, can you drop the space